Neos.Flow/Classes/ObjectManagement/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -513,6 +527,25 @@ public function buildPropertyInjectionCodeByConfigurationTypeAndPath(Configurati
         return ['$this->' . $propertyName . ' = ' . $preparedSetterArgument . ';'];
     }
 
+    /**
+     * Builds code which assigns the frontend of the specified cache into the given class property.
+     *
+     * @param Configuration $objectConfiguration Configuration of the object to inject into
+     * @param string $propertyName Name of the property to inject
+     * @param string $cacheIdentiier the configuration type of the injected property (one of the ConfigurationManager::CONFIGURATION_TYPE_* constants)
+     * @return array PHP code
+     */
+    public function buildPropertyInjectionCodeByCacheIdentifier(Configuration $objectConfiguration, string $propertyName, string $cacheIdentiier): array
+    {
+        $className = $objectConfiguration->getClassName();
+        $preparedSetterArgument = '\Neos\Flow\Core\Bootstrap::$staticObjectManager->get(\Neos\Flow\Cache\CacheManager::class)->getCache(\'' . $cacheIdentiier . '\')';
```

Comment:
```suggestion
     * @param string $cacheIdentifier the configuration type of the injected property (one of the ConfigurationManager::CONFIGURATION_TYPE_* constants)
     * @return array PHP code
     */
    public function buildPropertyInjectionCodeByCacheIdentifier(Configuration $objectConfiguration, string $propertyName, string $cacheIdentifier): array
    {
        $className = $objectConfiguration->getClassName();
        $preparedSetterArgument = '\Neos\Flow\Core\Bootstrap::$staticObjectManager->get(\Neos\Flow\Cache\CacheManager::class)->getCache(\'' . $cacheIdentifier . '\')';
```

and i also was wondering if `"'" . $bla . "'"` is good enough as we never validate that a cache identifier might not contain a `'` itself - but would work 99% of the time so fine ^^

Reply:
Regarding the "'" we do pretty much the same for settings nowadays. The current solution would be to make the cache identifier a valueObject but this is beyond the scope here. 

The suggested changes are applied